app/controllers/postsController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Application\Controllers\PostsController;
+
+use Application\Controllers\Controller;
+use Application\Models\PostsModel\PostsRepository;
+use Application\Core\Database\Database\DatabaseConnection;
+
+// to use requite here have to fbe fix.... still a issue
+require_once(ROOT . '/app/core/database/database.php');  // used for $connection
```

Comment:
You might need to do a general `require` of all files at the index.php

Reply:
It's what i did: i put all the requires in my config.php file which is loaded in the index.php file and it's ok

Reply:
.... but a can make a special file for it also....

Reply:
I resolve this issue by coding a special  autoload

`function my_autoloader($class)
{
    $delim = '\\';
    $myArray = explode($delim, $class);
    $newarray = array_slice($myArray, 1, -1);
    $delim = "/";
    $class2 = implode($delim, $newarray);
    $path = (ROOT . "/app/" . $class2 . ".php");

    if (file_exists($path)) {
        require $path;
    }
}
spl_autoload_register('my_autoloader');
`